src_code/util/defn.py

```diff
@@ -97,3 +98,19 @@ class ScriptTemplate(BaseModel):
 	script_log_path: str
 	cast_path:       str
 	engine_run_path: str
+
+
+class FlywheelJob(BaseModel):
+	"""
+	Represents some minimum properties for a flywheel job. Currently, it's only
+	being used for testing, so only properties relevant for testing are needed.
```

Comment:
NIT: Do you envision using it in the future for the casting?

I'm asking because if it is only used for testing, I think it should go somewhere in the `tests` folder.